stable/kong/ci/dbless-no-kic-external-declarative-config-values.yaml

```diff
@@ -0,0 +1,9 @@
+# CI test for testing dbless deployment without ingress controllers
+ingressController:
+  enabled: false
+env:
+  database: "off"
+postgresql:
+  enabled: false
+dblessConfig:
+  configMap: test-declarative-config
```

Comment:
In CI env, where will this ConfigMap come from?
You have not defined any fixture for this.
I don't know if this is even possible or not.

Reply:
I couldn't find any documentation which states that this is possible.
Please remove this test-case.